### DISCUSSION TITLE
Scenario "Tokyo": can't serve web file

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#25] [Scenarios]
Discussions about scenario [Tokyo](https://github.com/fduran/sadservers/tree/main/scenarios/tokyo)

**Comment by rachelsfba** [comment]
sshd public key authentication seems to be broken.  I cannot SSH into the VMs for this scenario, and have tried using the provided private key as well as the one associated with my profile.  I only have this problem with this particular scenario's VMs.

**Comment by fduran** [comment]
Hello @rachelsfba , thanks a lot for bringing this up. Direct SSH works but this is a Ubuntu VM (the other one left is Jakarta) and I'm not showing the right ssh instructions, you'd have to use the `ubuntu` user instead of `admin`, ie: `ssh -i ~/.ssh/your_priv_key ubuntu@<vm_pub_ip>`. Need to fix this.

**Reply by rachelsfba** [reply]
Logging in with the `ubuntu` user works for now, thanks!  I also realized after the fact that I probably should have posted this comment as a separate Issue instead of posting here in the Discussions area, so sorry about that.
